### PR TITLE
Multiple transactions

### PR DESCRIPTION
**Pull request by vkmita**
@ggrossman 

Tested this and it works with Faraday and Faraday follow_redirects middleware.

Watcha think?

### CODE REVIEW

**Comment by vkmita** [comment]
Ugh I can remove byebug I guess

**Comment by vkmita** [comment]
If this is OK we will have to update the README too

File: lib/raw_net_capture.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 
 class RawNetCapture < StringIO
```
Comment (ggrossman) [inline lib/raw_net_capture.rb:3]: Does it make sense to alter `RawNetCapture` in this way? `RawNetCapture` doesn't really make any assumptions about the underlying protocol, so not sure it really makes sense to talk about 'transactions' for it

Comment (ggrossman) [inline lib/raw_net_capture.rb:3]: (modifying `RawHTTPCapture` makes sense though)

File: lib/raw_net_capture.rb [hunk 2]
```diff
@@ -30,26 +44,27 @@ def initialize
...
-  def headers
-    separator = "\r\n\r\n"
-    raw_string = @raw_received.string
-    if headers_end_index = raw_string.index(separator)
-      raw_string[0...(headers_end_index + separator.length)]
-    else
-      raw_string
+    if @getting_response
+      @getting_response = false
+      @transactions << {
```
Comment (ggrossman) [inline lib/raw_net_capture.rb:61]: Should be some way to DRY this up, same hash is created above. Maybe a class for `RawHTTPTransaction` or something?

**Comment by vkmita** [comment]
Closing in favor of https://github.com/zendesk/raw_net_capture/pull/5

**Pull request closed without merging**